nncf/quantization/advanced_parameters.py

```diff
@@ -178,8 +205,12 @@ class AdvancedQuantizationParameters:
     disable_bias_correction: bool = False
 
     # Advanced Quantization parameters
-    activations_quantization_params: QuantizationParameters = field(default_factory=QuantizationParameters)
-    weights_quantization_params: QuantizationParameters = field(default_factory=QuantizationParameters)
+    activations_quantization_params: Union[QuantizationParameters, ConvertParameters] = field(
+        default_factory=QuantizationParameters
+    )
+    weights_quantization_params: Union[QuantizationParameters, ConvertParameters] = field(
```

Comment:
FP8 is represented not as the `*quantize` layer itself. Highly likely it will be presented as a usual `Convert` layer in the future.
In this case, naming `FP8QuantizationParameters` is weird and may confuse users since they would like to see some `*quantize` layers after using `*QuantizationParameters`.

Reply:
Just follow the hands 😄 
nncf.**quantize**, nncf.**Quantization**Mode, nncf.Advanced**Quantization**Parameters, activations_**quantization**_params

I am pretty much sure most of the users are not interested in what happens inside the model. But it would be better to be consistent on the API level.
BTW, can we just keep `QuantizationParameters` parameters?